<?xml version="1.0"?>
<launch>
	<!-- Аргументы для конфигурации -->
	<arg name="debug" default="true"/>
	<arg name="rate"  default="1.0"/>
	<!-- <arg name="pkg_path" default="$(find your_package)" /> -->
	<env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find manipulator_control)/rosconsole.conf"/>
	<param name="control_rate" value="$(arg rate)"/>
  
  <node pkg="rosbag" type="record" name="debug_log"
  	output="screen"
    args="-O $(find manipulator_control)/.bags/console.bag /rosout"/>

<!-- <node pkg="rosbag" type="record" name="debug_log" -->
<!--       args="-o $(find manipulator_control)/.bags/console.bag -e &quot;(/manipulator1/[_a-z]+)&quot;"/> -->

	<!-- Основной узел манипулятора -->
	<node pkg="manipulator_control" 
        type="manipulator_control_node" 
        name="manipulator_control_node_1" 
    output="screen">
    <param name="rosconsole_logger_level" value="DEBUG"/>
		<!-- Параметры узла -->
		<!-- <param name="rosconsole_logger_level" value="DEBUG"/> -->
		<!-- Переназначение топиков -->
		<!-- <remap from="/pipe_output"         to="/parameters/pipe"/> -->
		<remap from="/elbow_input"         to="/manipulator1/elbow_input"/>
		<remap from="/elbow_output"        to="/manipulator1/elbow_output"/>
		<remap from="/shoulder_input"      to="/manipulator1/shoulder_input"/>
		<remap from="/shoulder_output"     to="/manipulator1/shoulder_output"/>
		<remap from="/payload_output"      to="/manipulator1/payload_output"/>
		<remap from="/manipulator_command" to="/manipulator1/manipulator_command"/>
	</node>

</launch>
